jpegenc/subblocks/quantizer/quantiser.py

```diff
@@ -0,0 +1,154 @@
+"""The above module is the hardware implementation
+    of quantizer top module"""
+
+from myhdl import always_seq, block, always_comb
+from myhdl import Signal, concat, modbv, intbv
+
+from .ramz import ramz
+from .quantiser_core import (quantizer, QuantOutputStream,
+                             QuantInputStream, QuantConfig)
+
+
+class QuantIDataStream(object):
```

Comment:
My intention was a Buffer exists infront of the block. So that I can take data from the Buffer.